tests/unit_test/formats/ejp_manuscript_xml_test.py

```diff
@@ -469,6 +475,22 @@ def test_should_extract_manuscript_type(self):
                 == [MANUSCRIPT_TYPE_1]
             )
 
+        def test_should_extract_preprint(self):
+            result = _parse_xml_with_defaults(
+                _manuscript_xml([_version_node({
+                    **VERSION_1,
+                    'preprint': PREPRINT_1
+                })])
+            )
+            assert (
+                _versions_prop(result.versions, 'preprint')[0]
```

Comment:
If we expect this to be a list, should it be called `preprints`? Although I believe there can only be one (or none)? So maybe it would be preferable to change the implementation to extract it as an optional dict, rather than a list?

Reply:
I thought exporting this like a list to keep in an array structure will be tidy and easier to manage in BQ. Maybe I am missundestanding what you are mentioning?

Reply:
Currently it is like that in BQ:
```sql
SELECT manuscript_id, preprint 
FROM `elife-data-pipeline.ci.sample_manuscript_version_truncated`
WHERE manuscript_id = '92418'
```

Reply:
Yes having a struct around a `preprint` makes sense. But if it is called `preprint` I would expect it to be just the struct, not an array of struct.
Otherwise calling it `preprints` would make sense but then it would be slightly more difficult to use and it would suggest there could be more than one. (And there may also be the question what to do if there is more than one)

Reply:
I'm not sure if it's possible to have more than one, as the XML structure suggests it might not be feasible. Additionally, logically, it shouldn't exist.